src/main/java/com/google/devtools/build/lib/remote/blobstore/http/AbstractHttpHandler.java

```diff
@@ -78,6 +82,14 @@ protected void addCredentialHeaders(HttpRequest request, URI uri) throws IOExcep
     }
   }
 
+  protected void addUserAgentHeader(HttpRequest request) {
+    if (USER_AGENT_VALUE == null) {
+      USER_AGENT_VALUE =
+          "bazel/" + StringUtils.defaultIfEmpty(BlazeVersionInfo.instance().getVersion(), "dev");
```

Comment:
as discussed yesterday I believe we should try to be consistent between the gRPC protocol and the http protocol. So I'd argue if no version is provided just have plain `bazel` and if a version is provided have `bazel <version number>`.

Reply:
Also I don't think we use the `bazel/` notation anywhere else but typically just say `bazel <version>`. Is there a reason why we should use `/` in user agent?

Reply:
As discussed, this is a standard format, see: https://tools.ietf.org/html/rfc7231#section-5.5.3 and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent